Source/WebCore/rendering/RenderBox.h

```diff
@@ -696,6 +696,7 @@ class RenderBox : public RenderBoxModelObject {
 
     bool isGridItem() const { return parent() && parent()->isRenderGrid() && !isExcludedFromNormalLayout(); }
     bool isFlexItem() const { return parent() && parent()->isFlexibleBox() && !isExcludedFromNormalLayout(); }
+    bool isBlockLevelBox() const { return style().isDisplayBlockLevel() && !isExcludedFromNormalLayout(); }
```

Comment:
isExcludedFromNormalLayout is certainly not needed here.

Reply:
Removed